src/main.rs

```diff
@@ -26,6 +26,10 @@ struct Cli {
     )]
     require_device: bool,
 
+    #[cfg(target_os = "linux")]
+    #[clap(long, short = 'd', help = "The path of the video device to monitor, example: /dev/video0 (Linux only)")]
```

Comment:
Just a stylistic change for my personal preferences 😇 

```suggestion
    #[clap(long, short = 'd', help = "The path of the video device to monitor (e.g. `/dev/video0`) (Linux only)")]
```